SoomlaAndroidStore/src/com/soomla/store/data/StoreInfo.java

```diff
@@ -41,13 +41,17 @@
 import com.soomla.store.exceptions.VirtualItemNotFoundException;
 import com.soomla.store.purchaseTypes.PurchaseType;
 import com.soomla.store.purchaseTypes.PurchaseWithMarket;
+import com.soomla.store.purchaseTypes.PurchaseWithVirtualItem;
 import org.json.JSONArray;
 import org.json.JSONException;
 import org.json.JSONObject;
 
+import java.lang.IllegalArgumentException;
+import java.util.*;
```

Comment:
@EugeneButusov, you're not consistent in imports. If you use ".*", why do you use ".<Class>"?